.circleci/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:xenial
+
+USER root
+
+RUN apt-get update && apt-get install --yes \
+    git \
+    gcc \
+    build-essential \
+    autoconf \
+    automake \
+    libtool \
+    zlib1g-dev \
+    python2.7 \
+    python-dev \
+    python-pip \
+    python-tox
+
+RUN pip install coverage
+
+RUN git clone https://github.com/openstack/liberasurecode.git && \
```

Comment:
Comment why lib sucks on Ubuntu

Reply:
The version installed on ubuntu by "apt-get install liberasurecode-dev" is 1.1.0-3 on xenial, which contains a bug related to dependencies on internal openstack libraries libisal and libshss.

This bug is corrected in versions >= 1.2.1, but apt-get is not aware of them:
"E: Version '1.3.0' for 'liberasurecode-dev' was not found"

Building and installing liberasurecode-dev from source solves the problem.


More information:

https://stackoverflow.com/questions/40844556/compile-liberasurecode-missing-libshss-so

https://ask.openstack.org/en/question/93267/unable-to-start-swift-proxy-liberasurecode-missing-libshssso/